Custom Status/dist/metadata.json

```diff
@@ -9,7 +9,7 @@
         "tr": "Custom Status, Discord'un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak istediğiniz resmi seçme olanağı dışında bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz."
     },
     "url": "eggsy.codes",
-    "regExp": "^.*?premid.*?custom.*?status.*?$",
+    "regExp": "(eggsy[.]codes(?!\\\/projects\\\/premid\\\/custom-status))",
```

Comment:
dm'd you on discord about these being the wrong ones.

Reply:
We'll see...